arch/posix/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
-if (CONFIG_64BIT)
-	# some gcc versions fail to build without -fPIC
-	zephyr_compile_options(-m64 -fPIC)
-	zephyr_link_libraries(-m64)
-else ()
-	zephyr_compile_options(-m32)
-	zephyr_link_libraries(-m32)
-endif ()
+# @Intent: Set necessary compiler & linker options that apply globally for OS
```

Comment:
Minor: OS & ARCH are very ambiguous in this context.

I would change this comments to say something like:
"Set necessary compiler & linker options for this host OS"
"Set necessary compiler & linker options for this host architecture"
"Set necessary compiler & linker options for this specific host OS & architecture"

Reply:
Done!